src/lib/Components/Consignments/Screens/Overview.tsx

```diff
@@ -67,9 +79,11 @@ export default class Info extends React.Component<Props, ConsignmentSetup> {
   updateLocation = (city: string, state: string, country: string) =>
     this.updateStateAndMetaphysics({ location: { city, state, country } })
 
-  updateStateAndMetaphysics = (state: any) => this.setState(state, this.updateMetaphysics)
+  updateStateAndMetaphysics = (state: any) => this.setState(state, this.updateLocalStateAndMetaphysics)
+
+  updateLocalStateAndMetaphysics = async () => {
+    this.saveStateToLocalStorage()
```

Comment:
The method says it updates State and Metaphysics, but it seems it just set the item on the `AsyncStorage` (update the local store). I'm mainly curious if it should also be updating metaphysics (or if it does some where and I missed that). If it is indeed just updating the local store do it does it need to be called in the `submitFinalSubmission` on like 97?

Reply:
The lines underneath ( 73-77 ) update Metaphysics (`updateSubmission` and `createSubmission`) - they're still around 👍

Reply:
gotcha!